src/test/java/org/reaktivity/specification/nukleus/kafka/streams/FetchIT.java

```diff
@@ -580,4 +557,15 @@ public void shouldReceiveTombstoneAfterCompactedTombstone() throws Exception
         k3po.notifyBarrier("ROUTED_CLIENT");
         k3po.finish();
     }
+
+    @Test
+    @Specification({
+        "${scripts}/filter.not.equals.header/client",
+        "${scripts}/filter.not.equals.header/server"})
```

Comment:
This should follow naming convention `filter.not.header` rather than `filter.not.equals.header` because the other scenarios are named `filter.header` etc rather than `filter.equals.header` etc.